clang/lib/Sema/SemaDecl.cpp

```diff
@@ -9760,8 +9760,8 @@ Sema::ActOnFunctionDeclarator(Scope *S, Declarator &D, DeclContext *DC,
   if (getLangOpts().CPlusPlus) {
     // The rules for implicit inlines changed in C++20 for methods and friends
     // with an in-class definition (when such a definition is not attached to
-    // the global module).  User-specified 'inline' overrides this (set when
-    // the function decl is created above).
+    // the global module). This does not affect declarations, that are already
+    // inline, for example due being declared `inline` or `consteval`
```

Comment:
```suggestion
    // the global module). This does not affect declarations that are already
    // inline (whether explicitly or implicitly by being declared constexpr, etc)
```

Reply:
Updated the comment and added entry in bug fixes changes.